src/views/NewDraftWizard.js

```diff
@@ -50,7 +52,7 @@ const getStepContent = (wizardStep: number, createFolderFormRef: CreateFolderFor
     case 2:
       return <WizardShowSummaryStep />
     default:
-      throw new Error("Unknown step")
+      throw new Error("Unknown step") //THIS ERROR IS NOT CATCHED
```

Comment:
could this be solved as part of this PR, maybe 404 would make sense with the meaning: "a step that does not exist is being accessed."

Reply:
we can redirect to `Error404` page

Reply:
Maybe, I made a separate issue about this: https://github.com/CSCfi/metadata-submitter-frontend/issues/220 .

Reply:
i would like this part of this PR, `401` is for `Unauthorized` which does not make sense as the user is not trying to access something (s)he does not have permissions for but something that does not exist. So 404 makes more sense.

also don't leave comments in the code, if something is not finished put in issues, leave the code clean.

Reply:
Step now defaults to 404